src/test/scala/scraml/libs/TapirSupportSpec.scala

```diff
@@ -131,6 +132,59 @@ final class TapirSupportSpec
       )
     }
 
+    "generate enumeration types with default variant" in {
+      val params = ModelGenParams(
+        new File("src/sbt-test/sbt-scraml/simple/api/simple.raml"),
+        new File("target/scraml-tapir-test"),
+        "scraml",
+        FieldMatchPolicy.Exact(),
+        DefaultTypes(),
+        librarySupport = Set(CirceJsonSupport(), TapirSupport("Endpoints")),
+        formatConfig = None,
+        generateDefaultEnumVariant = Some("Unknown")
+      )
+
+      val generated = ModelGenRunner.run(DefaultModelGen)(params).unsafeRunSync()
+      val enumCompanion =
+        generated.files.find(_.source.name == "SomeEnum").flatMap(_.source.companion)
+
+      enumCompanion.map(_.toString.stripTrailingSpaces) should be(
+        Some(
+          """object SomeEnum {
+            |  case object A extends SomeEnum
+            |  case object B extends SomeEnum
+            |  case class Unknown(value: String)
```

Comment:
Should `Unknown` not extend `SomeEnum`? 🤔

Reply:
Right. I'll fix that